environments/staging/values.yaml

```diff
@@ -0,0 +1,29 @@
+image:
+  repository: 719139164272.dkr.ecr.us-west-2.amazonaws.com/gtfs-to-html-service
+
+replicaCount: 1
+
+healthcheck:
+  path: /health
+
+port: 8080
+
+envVars:
+  PORT: 8080
+  HTTP_POST_WHITELIST: 52.40.192.144,54.187.52.190
```

Comment:
this just covers posts from new staging & prod envs, right?

Reply:
these are only staging addresses